src/BuildScriptGenerator/BaseBashBuildScript.sh.tpl

```diff
@@ -104,7 +106,9 @@ then
 	{{ for excludedDir in DirectoriesToExcludeFromCopyToBuildOutputDir }}
 	excludedDirectories+=" --exclude {{ excludedDir }}"
 	{{ end }}
-	rsync -rtE --links $excludedDirectories . "$DESTINATION_DIR"
+
```

Comment:
should we remove that copying to a separate directory in our tests to see if this fix helps those issues as well?

Reply:
yes, good point. We should now be able to fix those issues as well. I will make that change as part of this PR.